ACCESSIBILITY.md

```diff
@@ -0,0 +1,225 @@
+```html
+<div class="o-forms-field" role="group" aria-labelledby="example-group-title" aria-describedby="example-group-info">
+	<span class="o-forms-title">
+		<span class="o-forms-title--main" id="example-group-title">Group Title</span>
+		<span class="o-forms-title--prompt" id="example-group-info">Group Info</span>
+	</span>
+	<span class="o-forms-input o-forms-input--checkbox">
+		<label>
+			<input type="checkbox" name="default" value="checkbox1" checked aria-label="Checkbox 1">
+			<span class="o-forms-input__label" aria-hidden="true">Checkbox 1</span>
+		</label>
+		<label>
+			<input type="checkbox" name="default" value="checkbox2" checked aria-label="Checkbox 2">
+			<span class="o-forms-input__label" aria-hidden="true">Checkbox 2</span>
+		</label>
+		<label>
+			<input type="checkbox" name="default" value="checkbox3" checked aria-label="Checkbox 3">
+			<span class="o-forms-input__label" aria-hidden="true">Checkbox 3</span>
+		</label>
+	<span>
+</div>
+```
+
+## Form Validation
+
+_Phew_. Now that we've covered static attributes for our form elements, let's jump into dynamic form validation. There are many approaches to validating in an accessible way, and none are infallible due to the nature of ATs, but we'll cover what we've considered so far.
+
+#### Native browser validation
+
+Browsers have native form validation. This could mean different implementation details for different browsers, but overall, a browser will check input fields in a form against the [ValidityState interface](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState), which checks for pattern matches, whether input is present in a required field, among a few other conditions.
+
+This means that invalid inputs will recieve messages tailored to describe the condition that has _not_ been met. These messages are presented in the language that the user has chosen for their browser. In addition to that, the browser will shift focus onto the first invalid input. Depending on the browser, the AT will read out the generated error message. 
+
+#### Non native validation
+
+We do _not_ style the tool tip that the browser's message appears in, so frequently forms are given the `novalidate` attribute, which doesn't run any native validation on submit. This means that the messages can be overriden and styled according to the FT's design and the beahviour determined by the product's needs.
+
+Primarily because of the above, `o-forms` doesn't make any decisions about validation in terms of behaviour.
+
+#### `aria-live` regions
+
+At its core, `aria-live` is an attribute that can be set to an area of a document to denote that the AT should pick up changes to that area. It is possible to determine what kind of changes it should recognise (additions, changes to text, etc), how forceful it should be in informing the user of the change (polite, assertive), alongside a few more advanced features.
```

Comment:
I guess this section needs updating since your work on validation and error messages? Since it handles aria-live (I'm assuming) do we need to talk about it in such detail?

Reply:
it needs to be rewritten, really. On it 🏃‍♀